cmd/trayicon.go

```diff
@@ -22,8 +23,10 @@ var trayiconCmd = &cobra.Command{
 	Use:   "trayicon",
 	Short: "Display the status of the checks in the system tray",
 	Run: func(cc *cobra.Command, args []string) {
+		lockDir, _ := shared.UserHomeDir()
 
-		lockFile, err := singleinstance.CreateLockFile("paretosecurity-tray.lock")
+		// Ensure lock directory exists
+		lockFile, err := singleinstance.CreateLockFile(filepath.Join(lockDir, ".paretosecurity-tray.lock"))
```

Comment:
I'm gonna assume this file is hidden from user, as I am afraid if it is not, someone will trash it.

Reply:
Hidden yes. On windows, this defaults to home folder, which is actually not seen by the users.